source/templates.tex

```diff
@@ -2472,9 +2472,11 @@
 \end{codeblock}
 \end{example}
 \begin{note}
+%% Ed. note: Using \iref would cause an overflowing \hbox; changing
+%% the sentence would lose the parallelism to [temp.mem] p5.
 Because the explicit template argument list follows the function template
-name, and because conversion member function templates and constructor
-member function templates are called without using a function name,
+name (\ref{temp.arg.explicit}), and because conversion member function templates\iref{class.conv.fct}
+are named without using a function name,
 there is no way to provide an explicit template argument list for these
 function templates.
```

Comment:
The wording that remains here is wrong, isn't it? The reason you can't provide an explicit template argument list to a conversion function is simply because grammar productions for *template-id* do not include a production for a *conversion-function-id* followed by a *template-argument-list*. (The reason for *that* is presumably the visual ambiguity between template arguments of the conversion function and template arguments of its type, combined with there being no way to write `operator T </*T=*/int>`, since you can't name `T`.) But it seems simply wrong to say that conversion member function templates are called without using a function name; they can be.

Reply:
I don't like these "because" sentences in any case.